Dockerfile

```diff
@@ -1,14 +1,20 @@
-FROM amazonlinux:2018.03.0.20180827
+FROM amazonlinux:2.0.20200722.0
 
-ARG STACK_VERSION=2.3.1
-ENV \
-  PATH=/root/.local/bin:$PATH \
-  STACK_ROOT=/stack-root
+# GHC version should match the LTS version, otherwise the path will not have the correct ghc version
+ARG GHC_VERSION=8.8.4
+ARG LTS_VERSION=lts-16.15
+ARG STACK_VERSION=2.5.0.1
 
-RUN \
-  set -o xtrace && \
-  yum update -y && \
-  yum install -y \
+ARG ITPROTV_ROOT=/home/itprotv
+ARG STACK_ROOT=$ITPROTV_ROOT/.stack
+
+ENV LANG=C.UTF-8 \
+  LC_ALL=C.UTF-8 \
```

Comment:
Why not `en_US.UTF-8`?

Reply:
Copied it from: https://github.com/commercialhaskell/stackage/blob/master/automated/dockerfiles/lts-16.12/Dockerfile there has to be some reason for it.